docs/source/complex_numbers.rst

```diff
@@ -0,0 +1,188 @@
+--------------
+
+Currently, there is very minimal linear algebraic operation support for complex tensors.
+We currently support :func:`torch.mv`, :func:`torch.svd`, :func:`torch.qr`, and :func:`torch.inverse`
+(the latter three are only supported on CPU). However we are working to add support for more
+functions soon: :func:`torch.matmul`, :func:`torch.solve`, :func:`torch.eig`, :func:`torch.eig`,
+:func:`torch.symeig`. If any of the other ops would help your use case, please search if an issue has
+already been filed (https://github.com/pytorch/pytorch/issues?q=is%3Aopen+is%3Aissue+label%3A%22module%3A+named+tensor%22)
+and if not, file one (https://github.com/pytorch/pytorch/issues/new/choose).
+
+Serialization
+-------------
+
+Complex Tensors can be serialized, allowing data to be saved as complex values.
+
+::
+
+     >>> torch.save(y, 'complex_tensor.pt')
+     >>> torch.load('complex_tensor.pt')
+     tensor([ 0.6125-0.1681j, -0.3773+1.3487j, -0.0861-0.7981j])
+
+
+Autograd
+--------
+
+PyTorch supports Autograd for Complex Tensors.
+
+1. :func:`torch.functional.backward` can be used for holomorphic :math:`C -> C` functions.
+   For non-holomorphic functions, the gradient is evaluated as if it were holomorphic.
+2. :func:`torch.functional.backward` can be used to optimize :math:`C -> R` functions, like
+   real-values loss functions of complex parameters :math:`x` by taking steps in the direction
+   of conjugate of :math:`x.grad`.
+3. mention the current behavior of backward for spectral ops?
+
+For more details, check out the Autograd note :ref:`complex_autograd-doc`.
+
+::
+
+     x = torch.randn(2, 2, 2, dtype=torch.double, requires_grad=True)>>> x = torch.randn(2, 2, 2, dtype=torch.double, requires_grad=True)
+     >>> y = x.detach().requires_grad_(True)
+     >>> x0 = x.clone()
+     >>> x1 = torch.view_as_complex(x0)
+     >>> x2 = torch.view_as_real(x1)
+     >>> x2.mul_(2)
+     tensor([[[ 4.2425, -0.1076],
+          [ 3.2731,  2.3156]],
+
+          [[ 4.1179,  0.7358],
+          [-1.7711, -0.4389]]], dtype=torch.float64,
+          grad_fn=<ViewAsRealBackward>)
+     >>> x2.sum().backward()
+     >>> y0 = y.clone()
+     >>> y0.mul_(2)
+     tensor([[[ 4.2425, -0.1076],
+          [ 3.2731,  2.3156]],
+
+          [[ 4.1179,  0.7358],
+          [-1.7711, -0.4389]]], dtype=torch.float64, grad_fn=<MulBackward0>)
+     >>> y0.sum().backward()
+     >>> x.grad
+     tensor([[[2., 2.],
+          [2., 2.]],
+
+          [[2., 2.],
+          [2., 2.]]], dtype=torch.float64)
+     >>> y.grad
+     tensor([[[2., 2.],
+          [2., 2.]],
+
+          [[2., 2.],
+          [2., 2.]]], dtype=torch.float64)
+
+We do not support the following subsystems:
+
+Quantization
+
+JIT
+
+Sparse Tensors
+
+Distributed
+
+Multiprocessing
+
+If any of these would help your use case, please search if an issue has already been filed (https://github.com/pytorch/pytorch/issues?q=is%3Aopen+is%3Aissue+label%3A%22module%3A+named+tensor%22)
```

Comment:
Wrong link as well I think

Reply:
updated

Reply:
Still looks wrong...

Reply:
yeah haven't pushed the local changes yet sorry 😅